src/MobileRedirection.php

```diff
@@ -71,58 +70,16 @@ public function filter_default_options( $defaults ) {
 	 * @return array Sanitized options.
 	 */
 	public function sanitize_options( $options, $new_options ) {
-		$options[ Option::MOBILE_REDIRECT ] = (
-			isset( $new_options[ Option::MOBILE_REDIRECT ] )
-			&&
-			rest_sanitize_boolean( $new_options[ Option::MOBILE_REDIRECT ] )
-		);
+		if ( isset( $new_options[ Option::MOBILE_REDIRECT ] ) ) {
```

Comment:
The backend form submission was resetting this back to false because it was assuming the `mobile_redirect` option would be in the form data. Now that that `mobile_redirect` is among the options updated via REST, we need this check to make sure it's ignored if necessary.